src/base/castleutils.pas

```diff
@@ -42,8 +42,6 @@
   Initialization of this unit does some generally-useful things:
 
   @unorderedList(
-    @item(Calls Randomize (so that you never forget about it).)
-
     @item(Sets DecimalSeparator to '.'.
```

Comment:
As mentioned in the PR comment, I would prefer to revert this change, and keep using standard `Random` and `Randomize` throughout the engine. Unless there's a good reason... but so far I don't see one -- the standard `Random` worked good enough (fast enough, and is random enough) for my purposes so far. The `CastleRandom` is great to have as an alternative, but for now I think I prefer to not make it a default...